.github/workflows/CI.yml

```diff
@@ -42,6 +42,31 @@ jobs:
       run: |
         make doc
 
+  arkouda_python_portability:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Discussed on chat, but I'll see if there's an alias for the latest python and maybe an easier way to maintain this list going forward instead of having to manually add new versions.

Reply:
Added `'3.x'` as an alias for latest. 

Looks like there no way to say "test all minor versions >= 3.7", so I think this list will have to be manually maintained.